tests/misc_env/install_prereq.py

```diff
@@ -295,20 +304,11 @@ def enable_rhel_eus_rpms(ceph, distro_ver):
     ceph.exec_command(sudo=True, cmd="yum clean all", long_running=True)
 
 
-def registry_login(ceph, distro_ver):
+def registry_login(ceph, distro_ver, cloud_type="openstack"):
```

Comment:
Do we need cloud_type here? looks like this variable is not used in this method

Reply:
It was an intermediate change and is no longer require as the logic has been modified.